packages/sdk/src/sdk-blockchains/tezos/bid.ts

```diff
@@ -75,6 +75,8 @@ export class TezosBid {
 			payoutsSupport: PayoutsSupport.MULTIPLE,
 			supportedCurrencies: getSupportedCurrencies(),
 			baseFee: parseInt(this.provider.config.fees.toString()),
+			getConvertableValue: null as any,
```

Comment:
we have special utility for cases when something is not implemented yet
it shouldn't use `as any` in this case
it should throw error